lib/menu.rb

```diff
@@ -0,0 +1,31 @@
+class Menu
+
+  def initialize
+    @menu_list = {:steak => 15, :chips => 5, :pizza =>7,
+    :lasagna =>12, :chicken => 10, :sushi => 14}
+  end
+
+  def food
+    menu_string = "\nMenu\n"
+    @menu_list.each {|key,value|
+      menu_string = menu_string + "#{key}, #{value}\n"
+    }
+    menu_string = menu_string + "To place an order please reply with the food item and quantity in the following format (example): \n  pizza 3, steak 1, chips 2 ... etc."
```

Comment:
Metrics/LineLength: Line is too long. [170/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SelfAssignment: Use self-assignment shorthand `+=`. (https://github.com/bbatsov/ruby-style-guide#self-assignment)